nuklear.h

```diff
@@ -15767,6 +15767,7 @@ nk_panel_begin(struct nk_context *ctx, const char *title, enum nk_panel_type pan
                 &style->window.header.close_button, in, style->font) && !(win->flags & NK_WINDOW_ROM))
             {
                 layout->flags |= NK_WINDOW_HIDDEN;
+                layout->flags |= NK_WINDOW_CLOSED;
```

Comment:
Yea, this seems right. I actually think the `layout->flags |= NK_WINDOW_HIDDEN;` is wrong and should be removed as well.

Also this change needs to be done in [src/nuklear_panel.c](https://github.com/Immediate-Mode-UI/Nuklear/blob/master/src/nuklear_panel.c) so that the `paq` script does not override your change.

Reply:
After applying these changes, I got an assert in some other locations.